model/Cargo.toml

```diff
@@ -21,8 +21,8 @@ serde_repr = { default-features = false, version = "0.1" }
 tracing = { default-features = false, version = "0.1" }
 
 [dev-dependencies]
-criterion = "0.3"
-serde_json = { default-features = false, features = ["alloc"], version = "1" }
+criterion = { default-features = false, version = "0.3" }
+serde_json = { default-features = false, features = ["std"], version = "1" }
```

Comment:
Why switch to `std`?

Reply:
alloc should only be used in libraries with `#![no_std]` support. (and because other deps use std instead of alloc)

Reply:
We can revisit this if we plan to enable `#![no_std]` at some point